client/build/manifest.json

```diff
@@ -0,0 +1,15 @@
+{
+  "short_name": "React App",
+  "name": "Create React App Sample",
```

Comment:
You need to remove the entier client/build folder.

Reply:
I fixed it.